packages/ui-components/src/components/text-field/text-field.config.ts

```diff
@@ -7,6 +7,8 @@ export const NUMERIC_TEXT_INPUT_MASK_CONFIG: Inputmask.Options = {
 	rightAlign: false,
 	showMaskOnHover: false,
 	showMaskOnFocus: false,
+	clearMaskOnLostFocus: false,
+	placeholder: ' ',
```

Comment:
image that I provide a placeholder for the input, with this change what gonna happen?